lib/spotlight/dor/indexer.rb

```diff
@@ -40,67 +24,126 @@ class Indexer < GDor::Indexer
     before_index :add_series
     before_index :mods_cartographics_indexing
 
-    def solr_client
-      @solr_client
-    end
+    attr_reader :solr_client
```

Comment:
Yeah, we're overriding it from https://github.com/sul-dlss/gdor-indexer/blob/415797f9b79642d3e49649e9d7202d54c7b1c50a/lib/gdor/indexer.rb#L68

I assumed this would be identical to the existing code, but let me double-check that, and add a comment to remind ourselves in the future.